MPU/src/mpuCanHandlers.cpp

```diff
@@ -58,8 +58,12 @@ void bmsCurrentLimits_cb(const CAN_message_t &msg)
 void motorMotion_cb(const CAN_message_t &msg)
 {
     //angular motor speed is found at bytes 2 and 3
-    int16_t motorSpeed = ((msg.buf[3] << 8) | msg.buf[2]) / 10;
-    motorController.setMotorSpeed(motorSpeed);
+    int16_t motorSpeed = (msg.buf[3] << 8) | msg.buf[2];
+    if (motorSpeed >> 15 == -1) {
+        motorSpeed = motorSpeed - (1 << 16);
+    }
+
+    motorController.setMotorSpeed(-motorSpeed);
```

Comment:
We are multiplying by -1 because the motor is technically spinning backward when we go forward, right?

Reply:
uhhh yes?

Reply:
yep